src/server/src/proxy.ts

```diff
@@ -473,7 +430,7 @@ function updateTrackedAccounts() {
 }
 
 // Log function
-function logThis(str: string, level: LogLevel) {
+function logThis(str: string, level: LogLevel, userSettings: UserSettings = defaultUserSettings) {
```

Comment:
This one is a bit tricky, do you think a lot of users have a specific user log setting? :thinking: If they do we'd have to inject that for every log statement here..

Reply:
I think we can skip that one. Have one log setting for all users for simplicity